ftlib/__init__.py

```diff
@@ -4,18 +4,36 @@
 import threading
 
 from .consensus.consensus_status import ConsensusStatus
-from .consensus.shared_storage import SharedStorage
 from .framework.framework_status import FrameworkStatus
 from .ftlib_status import FTAllReduceStatus, FTRebuildStatus
 
+# in the ftlib package, user is able to initialize the package
+# with specific consensus method adn framework
+
+
+framework_list = ["dummy_NCCL", "pytorch"]
+consensus_list = ["gossip", "shared_storage"]
```

Comment:
Move these to constant classes

Reply:
I suppose that means static member variable. Please check the new implementation.